src/uploader.js

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
If this is the only file in the library, would it make sense to ditch the index.js one-liner and to rename this to index.js?

And then, if index.js is the only file in the `src` folder, perhaps it would make sense to ditch the folder and just have `"main": "index.js",` in the `package.json` instead?

Reply:
I'm not sure why this was in code, we can change it.